deploy/base/secret.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: objectstorage-provisioner
+  namespace: objectstorage-provisioner-ns
+  labels:
+    app.kubernetes.io/part-of: container-object-storage-interface
+    app.kubernetes.io/component: provisioner
+    app.kubernetes.io/version: main
+    app.kubernetes.io/name: container-object-storage-interface-provisioner-sidecar
+type: Opaque
+stringData:
+  LISTEN_ADDRESS: 0.0.0.0:9000
```

Comment:
Who'll be listening here?

Reply:
@rrati might be able to shed some light on this